### PR TITLE
Updated source of main figures behind a feature flag

### PR DESCRIPTION
**Pull request by JoshSinghONS**
### What

Added a feature flag that when enabled changes where the main figures from employment and unemployment are taken from.

### How to review

LGTM. Run the service with and without the flag to see that without the flag it is business as usual and with the flag that it errors for now. 

### Who can review

Anyone but probs Nick.

### CODE REVIEW

File: homepage/homepage.go [hunk 1]
```diff
@@ -99,14 +101,26 @@ func init() {
...
+		mainFigureMapCopy.uris = []string{"/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/timeseries/s2pw/lms"}
+		mainFigureMapCopy.trendURI = "/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/timeseries/s2qm/lms"
+		mainFigureMap["emp"] = mainFigureMapCopy
+
+		mainFigureMapCopy = mainFigureMap["unemp"]
+		mainFigureMapCopy.uris = []string{"/employmentandlabourmarket/peoplenotinwork/unemployment/timeseries/s2pu/lms"}
+		mainFigureMapCopy.trendURI = "/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/timeseries/s2qk/lms"
+		mainFigureMap["unemp"] = mainFigureMapCopy
+	}
+
```
Comment (mr-nick17) [inline homepage/homepage.go:123]: I would have preferred this in a distinct `if/else` block...

**TL;DR**
- It would be easier to read 
- Easier to remove the old employment stats later on
- Less reliant on being on a certain order i.e. you need lines 96-98 & 107-109 set first 
- Would save on rewriting to the same variable i.e. instantiated on line 113 and reused on 118. 

```go
if cfg.EnableUpdatedMainFigures {
		// Employment
		mainFigureMap["emp"] = MainFigure{
			uris:               []string{"/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/timeseries/s2pw/lms"},
			trendURI:           "/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/timeseries/s2qm/lms",
			datePeriod:         mapper.PeriodMonth,
			data:               zebedee.TimeseriesMainFigure{},
			differenceInterval: mapper.PeriodYear,
		}

		// Unemployment
		mainFigureMap["unemp"] = MainFigure{
			uris:               []string{"/employmentandlabourmarket/peoplenotinwork/unemployment/timeseries/s2pu/lms"},
			trendURI:           "/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/timeseries/s2qk/lms",
			datePeriod:         mapper.PeriodMonth,
			data:               zebedee.TimeseriesMainFigure{},
			differenceInterval: mapper.PeriodYear,
		}
	} else {
		// Employment
		mainFigureMap["emp"] = MainFigure{
			uris:               []string{"/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/timeseries/lf24/lms"},
			trendURI:           "/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/timeseries/fux7/lms",
			datePeriod:         mapper.PeriodMonth,
			data:               zebedee.TimeseriesMainFigure{},
			differenceInterval: mapper.PeriodYear,
		}

		// Unemployment
		mainFigureMap["unemp"] = MainFigure{
			uris:               []string{"/employmentandlabourmarket/peoplenotinwork/unemployment/timeseries/mgsx/lms"},
			trendURI:           "/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/timeseries/fuu8/lms",
			datePeriod:         mapper.PeriodMonth,
			data:               zebedee.TimeseriesMainFigure{},
			differenceInterval: mapper.PeriodYear,
		}
	} 
```

**Review by mr-nick17 — APPROVED** [review]
Will approve so this isn't blocking urgent work but will need looking at again

**Review by mr-nick17 — APPROVED** [review]
